### DISCUSSION TITLE
Update the doc please

### DISCUSSION THREAD

**Discussion by ManiaJack** [axios/axios#5429] [General]
We have a lot of new feature and breaking changes after v1.x, but the official doc is still under maybe v0.27. Some configuration is not applicable now like `paramsSerializer`.

As a famous and wildly used package, the doc is very important. So please please update the docs. Thx~

**Comment by lucapollani** [comment]
Hi, 
I agree with your request, I've met same differences.

The README seems more reliable and aligned.

**Comment by jasonsaayman** [comment]
Updated
